zebra/zebra_vxlan.c

```diff
@@ -871,10 +871,10 @@ static void zvni_print_neigh_hash(struct hash_bucket *bucket, void *ctxt)
 		if (json_vni == NULL) {
 			if ((wctx->flags & SHOW_REMOTE_NEIGH_FROM_VTEP) &&
 			    (wctx->count == 0))
-				vty_out(vty,
-					"%*s %-6s %-8s %-17s %-21s\n",
+				vty_out(vty, "%*s %-6s %-8s %-17s %-21s %s\n",
```

Comment:
Can you explain what is broken and what are the issue that was fixed as part of Json ?

Reply:
This particular line of change just adds the new column name - "Seq #'s" to the output. The actual issue that was fixed, is described in my comment below.